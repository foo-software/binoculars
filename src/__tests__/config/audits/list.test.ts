import auditList from '../../../config/audits/list';

describe('list audit', () => {
  it('should match snapshot', () => {
    expect(auditList('en')).toMatchSnapshot();
  });
});
