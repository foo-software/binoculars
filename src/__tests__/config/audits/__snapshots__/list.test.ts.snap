// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list audit should match snapshot 1`] = `
Object {
  "description": "Screen readers have a specific way of announcing lists. Ensuring proper list structure aids screen reader output. [Learn more](https://web.dev/list/).",
  "title": "Lists contain only \`<li>\` elements and script supporting elements (\`<script>\` and \`<template>\`).",
}
`;
